src/docs/product/performance/web-vitals.mdx

```diff
@@ -4,7 +4,9 @@ sidebar_order: 6
 description: "Learn more about Web Vitals in Sentry and how they give you a better understanding of your application's health."
 ---
 
-Frontend transactions may contain web vitals information if the browser provides them. With this view, you'll be able to examine a summary of the web vitals.
+[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and the sum of shifting rendering processes. Each data point provides insights about the overall performance of your application.
+
+The Sentry JavaScript SDK collects web vitals information from the browser (where supported) and adds that information to frontend transactions. These vitals are then summarized in several graphs for a quick overview of how each frontend transaction is performing for your users.
```

Comment:
Is there a reason we're focusing this solely on the JS SDK? If it's a function of only that SDK, then that's fine and we should say that. Otherwise, we can rephrase to be more comprehensive, such as "The browser-based SDKs" or something similar. Let me know if you need help with phrasing @jeremybourque

Reply:
my understanding is that React etc are classified under the JS SDK, but AIUI it wouldn't be wrong to say "browser SDK" and I agree that it would probably reduce confusion for people thinking "wait does React count or not". then we could probably also drop "from the browser".